src/providers/voice/twilio.js

```diff
@@ -0,0 +1,73 @@
+/* @flow */
+import fetch from '../../util/request'
+import FormData from 'form-data'
+// Types
+import type {VoiceRequestType} from '../../models/notification-request'
+
+export default class VoiceTwilioProvider {
+  id: string = 'voice-twilio-provider'
+  accountSid: string
+  apiKey: string
+
+  constructor ({accountSid, authToken}: Object) {
+    this.accountSid = accountSid
+    this.apiKey = Buffer.from(`${accountSid}:${authToken}`).toString('base64')
+  }
+
+  /*
+   * Note: 'type', 'nature', 'messageClass' are not supported.
+   */
+  async send (request: VoiceRequestType): Promise<string> {
+    const {
+      from,
+      to,
+      url,
+      method,
+      fallbackUrl,
+      fallbackMethod,
+      statusCallback,
+      statusCallbackMethod,
+      statusCallbackEvent,
+      sendDigits,
+      machineDetection,
+      machineDetectionTimeout,
+      timeout,
+      recordingChannels,
+      recordingStatusCallback,
+      recordingStatusCallbackEvent
+    } = request
+    const form = new FormData()
+    form.append('From', from)
+    form.append('To', to)
+    form.append('Url', url)
+    if (method) form.append('Method', method)
+    if (fallbackUrl) form.append('FallbackUrl', fallbackUrl)
+    if (fallbackMethod) form.append('FallbackMethod', fallbackMethod)
+    if (statusCallback) form.append('StatusCallback', statusCallback)
+    if (statusCallbackMethod) form.append('StatusCallbackMethod', statusCallbackMethod)
+    if (statusCallbackEvent) form.append('StatusCallbackEvent', statusCallbackEvent)
```

Comment:
The `form-data` lib does not accept arrays, is it ok if I change the code by:
```js
if (statusCallbackEvent) {
  statusCallbackEvent.forEach((event) => form.append('StatusCallbackEvent', event))
}
```
?

Reply:
Yes another good call. Looks like the docs for that is here: https://www.twilio.com/docs/api/voice/making-calls#statuscallback-parameter